src/main/java/base/DesignPattern.java

```diff
@@ -0,0 +1,17 @@
+package base;
+
+/**
+ * Instantiation can be done according to the following example. Using curly brackets to enter an
+ * array.
+ *
+ * @ DesignPattern(pattern = {DesignEnum.IMMUTABLE, DesignEnum.SINGLETON}) public class
```

Comment:
```suggestion
 * @ DesignPattern(pattern = {Pattern.IMMUTABLE, Pattern.SINGLETON}) public class
```
(or change the name of the pattern)